### ISSUE TITLE
Suggestion: options for split tunnel (DNS only/host IP only) and specific ports only 

### ISSUE THREAD

**Issue by zilexa** [WeeJeWel/wg-easy#11]
1.
Currently I use PiVPN, installed bare on my Ubuntu-based host homeserver. All other things run via Docker Compose. 
With PiVPN (during initial install) you can set a few client-config defaults, any client config that will be created will then in my example have:
```
DNS servers: 
192.168.88.2
Allowed IPs: 
192.168.88.0/24, ::/0
```
This way, the tunnel is only used to access the LAN (192.168.88.0/24) and for all DNS requests (my server runs Adguard Home in docker compose via network:host and Unboud in dockers bridge network).

May I suggest adding a ENV variable to set the default for Allowed IPs in the client config? You already provide the default DNS env variable. 

2.
One of the more difficult things to configure with Wireguard, is to _by default_, block access to any LAN service/port, except for service/port X,Y,Z: the services you actually want to be able to access remotely via VPN. For example: Jellyfin or Emby. 

Usecase: if I want to share Jellyfin or Emby with the parents or friends via wg-easy, I would only want to allow them to access port 8096, all other ports should be blocked by default. Otherwise, they would have full access to my server, other ports and other devices on the network. 

It would be absolutely amazing if this could be configured per client in the webUI (_block all ports, allow only ports ...).

**Comment by WeeJeWel** [comment]
Well, feel free to make a PR, but I want to keep this project simple and for newer users, so keep that in mind.

**Comment by zilexa** [comment]
I wouldn't know how, I am not a developer :) But in any case: consider any connected client will have full access to the host system. 

This means they have access to the wg-easy webUI and can create/delete new client configs for anyone. 
That could be of major concern for anyone... especially new users who do not realize this. 

I am also a newbie, I just came up with these suggestions because I have been thinking about how to expose Jellyfin, since it doesn't have 2FA, I would prefer to not expose it at all, only allow access via VPN. But configuring iptables is a bridge too far for me as rookie.

**Comment by WeeJeWel** [comment]
Users can set a `PASSWORD` environment variable to prevent access to the Web UI.

**Comment by danmed** [comment]
I would agree, whilst the simplicity is awesome, it would be nice to have some kind of split tunnel capability..

For now it does exactly what i'm after which is to allow my phone to hit my LAN easily when i'm out and about.. so thanks!

**Comment by WeeJeWel** [comment]
> I would agree, whilst the simplicity is awesome, it would be nice to have some kind of split tunnel capability..

I'm very good at saying _no_ to features <5% people use to keep it simple :)

**Comment by danmed** [comment]
> > I would agree, whilst the simplicity is awesome, it would be nice to have some kind of split tunnel capability..
> 
> I'm very good at saying _no_ to features <5% people use to keep it simple :)

hey man, fair enough.. it's your project.. just thought i would share my interest.

**Comment by WeeJeWel** [comment]
Absolutely welcome!
